src/PrestaShopBundle/Translation/Loader/DatabaseTranslationLoader.php

```diff
@@ -35,29 +35,40 @@ class DatabaseTranslationLoader implements LoaderInterface
 {
     /** @var EntityManagerInterface */
     protected $em;
-    
+
     /**
-     * 
      * @param EntityManagerInterface $em
      */
     public function __construct(EntityManagerInterface $em)
     {
         $this->em = $em;
     }
-    
+
     /**
      * {@inheritdoc}
      */
     public function load($resource, $locale, $domain = 'messages')
     {
         $lang = $this->em->getRepository('PrestaShopBundle:Lang')->findOneByLocale($locale);
-        $translations = $this->em->getRepository('PrestaShopBundle:Translation')->findBy(['lang' => $lang, 'domain' => $domain]);
+        $translationRepository = $this->em->getRepository('PrestaShopBundle:Translation');
+
+        $translations = $translationRepository->createNamedQuery('t')
+            ->where('t.lang =:lang')
```

Comment:
note that PHP CS Fixer dont complain about, this is probably not a psr requirement :)

fixed.